Cargo.toml

```diff
@@ -32,9 +32,10 @@ members = [
     "examples/tracing-grpc",
     "stress",
 ]
+resolver = "2"
 
 [profile.bench]
-# https://doc.rust-lang.org/cargo/reference/profiles.html#bench
+# https://doc.rust-lang.org/cargo/reference/profiles.html#debug
 # See function names in profiling reports.
-# 2/true is too much, 0 is not enough, 1 is just right for back traces
-debug = 1
+# full is too much, none is not enough, limited is just right for back traces
+debug = "limited"
```

Comment:
Looks like our MSRV doesn't yet support this.

Reply:
That's what I get for trying to change something small but not related to the real purpose of the PR. ;)

Reply:
Indeed, and in the same commit no less!